tasks/config.py

```diff
@@ -36,7 +36,7 @@ class Agent(BaseModel, extra=Extra.forbid):
         class Pulumi(BaseModel, extra=Extra.forbid):
             logLevel: Optional[int] = None
             logToStdErr: Optional[bool] = None
-            verboseProgressStreams: Optional[bool] = None  # noqa
+            verboseProgressStreams: Optional[bool] = None
```

Comment:
I added it there to not have failure when using a `~/.test_infra_config.yaml` with that key. The issue with that key is that there is no equivalent between what we use this flag for on `datadog-agent` with Pulumi API Stack manager and the flags that are available with `pulumi` cli. 
With the Pulumi API, when this flag is set to `false` we will only display resource deployment logs for resource that failed to be deployed.
With `pulumi up` CLI here it would only suppress progress points.

Reply:
Yes, I forced it into dropping the points progress, but I can revert and just skip the usage of this key. Changing it